pkg/network/common/egressip_test.go

```diff
@@ -120,9 +120,11 @@ func TestEgressIP(t *testing.T) {
 	eit, w := setupEgressIPTracker(t)
 
 	updateHostSubnetEgress(eit, &osdnv1.HostSubnet{
+		Subnet: "172.17.0.0/24",
 		HostIP: "172.17.0.3",
```

Comment:
`Subnet` is the SDN subnet assigned to the node, so it doesn't make sense for it to be overlapping `HostIP`... and yeah, it doesn't actually matter for the tests _currently_, but let's just get it right so no one has to change it again in the future.
`"10.128.0.0/23"` is an acceptable value to use instead.

(I guess I should have said that on the `pkg/network/common` tests too. And likewise in the `pkg/network/node` tests below.)